mobile/src/main/java/ch/epfl/sdp/mobile/ui/play/PlayScreen.kt

```diff
@@ -46,25 +50,64 @@ fun <M : ChessMatch> PlayScreen(
     ) { items(state.matches, key) { match -> Match(match, { state.onMatchClick(match) }) } }
   }
 
-  Box(modifier = modifier.fillMaxSize().padding(contentPadding)) {
+  Box(
+      modifier = modifier.fillMaxSize().padding(contentPadding),
+      contentAlignment = Alignment.BottomEnd) {
```

Comment:
It is a bad practice to have multiple two or more top-level composable calls without receivers (like the `Column` and `Box` calls here), since nothing indicates how the parent layout will organize these composables. Indeed, the layout will end up being different if the caller is a `Column` or a `Row` for instance.

Please wrap the FAB and the `Column` with content in a `Scaffold` instead.

Reply:
Fixed in [c721716](https://github.com/epfl-SDP/android/pull/226/commits/c721716bc39f817e2e9ca79024802da1c7f58b6b)